src/middleware/healthcheck-middleware.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import errors from 'restify-errors'
+
+function checkDatabase(req: any, res: any, next: any) {
+  if (process.env.BLOCK_RESPONSE !== 'true') {
```

Comment:
shouldn't this check for the opposite, i.e. `process.env.BLOCK_RESPONSE === 'true'` ?

Reply:
I probably mixed something up when renaming the variable, `process.env.BLOCK_RESPONSE` will be true iff the block number changed. Using 'block' in the name is a bit ambiguous, do you have some other suggestion?

Reply:
I agree, it's ambiguous. Perhaps `DATABASE_UNHEALTHY`? I know I was originally advocating having a flag for stopping responses for a generic reason, but that's an overkill I guess.